schema/image.go

```diff
@@ -65,7 +65,8 @@ func (i *BuildFormat) Set(value string) error {
 // BuildImageName builds a Docker image tag for build, push or deploy
 func BuildImageName(format BuildFormat, image string, version string, branch string) string {
 	imageVal := image
-	if strings.Contains(image, ":") == false {
+	splitImage := strings.Split(image, "/")
```

Comment:
I think I'd have used `strings.Index()` here in both cases, and used that for the operations.

If the tests pass, this is fine to keep as is.